FrontDesk/tests/UnitTests/Core/AggregatesEntities/AppointmentTests/Appointment_UpdateEndTime.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using AutoFixture;
+using FrontDesk.Core.Aggregates;
+using Xunit;
+
+namespace UnitTests.Core.AggregatesEntities.AppointmentTests
+{
+  public class Appointment_UpdateEndTime
+  {
+    private readonly DateTime _startTime = new DateTime(2021, 01, 01, 10, 00, 00);
+    private readonly Fixture _fixture = new Fixture();
+
+    [Fact]
+    public void SuccessWhenUpdateEndTime()
+    {
+      var scheduleId = Guid.NewGuid();
+
+      var appointment =
+        Appointment.Create(scheduleId, 1, 1, 1, _startTime, _startTime.AddHours(3), 1, 1, "Title Test");
```

Comment:
These literals you can get away with because:
- They're not repeated in the test (for assertions, etc)
- They're not repeated in the class
- They're not important to what is being tested

Reply:
That said, you might be able to just use `_fixture.Create` to get a working appointment instead of the `Appointment.Create` factory method. That's what Autofixture is good for - giving you instances of objects that work but where you don't really care what most of their properties are set to.